cpp/pagerank_module/algorithm/pagerank_algorithm.hpp

```diff
@@ -0,0 +1,166 @@
+/// @file
+
+#pragma once
```

Comment:
#ifndef guards and #pragma once are both used we need to decide on one.
I'd go with the pragma

Reply:
I agree.